doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -1,14 +1,14 @@
 (kubectl-plugin)=
 
-# Use kubectl Plugin (alpha)
+# Use kubectl Plugin (beta)
```

Comment:
Can the casing here be consistent, either `Use kubectl plugin (beta)` or `Use Kubectl Plugin (beta)`

Reply:
Done!